### ISSUE TITLE
Issue in Building on Windows

### ISSUE THREAD

**Issue by gaojindev** [sighthoundinc/SighthoundVideo#12]
I installed ninja, cmake, msys2, VS2022, LLVM
But when I run 'buildSV.sh' on VS2022 command prompt, it gives me error.
I can't see the log message because when I run buildSV.sh, it opens a new shell window and dies after a few seconds.
![image](https://github.com/sighthoundinc/SighthoundVideo/assets/20929679/976d9c74-79d9-4666-b870-23efdb54166e)
Above is error message I get when I run buildSV.sh on msys2.exe
I think the error occurs because the tar files in win directory are invalid

**Comment by w3sip** [comment]
The `tar` problem is likely because git lfs files were not pulled.
It also appears there is no cmake available.

**Comment by gaojindev** [comment]
Thanks for your answer.
I installed cmake and add cmake into path by set PATH=%PATH% /path/to/cmake/bin
Let me fix the cmake issue.
By the way, could you provide me tgz files?

**Comment by gaojindev** [comment]
Could you provide me tgz files- lib.tgz and tools.tgz in win directory?

**Comment by w3sip** [comment]
They are there - https://github.com/sighthoundinc/SighthoundVideo/tree/main/win. I'll have to repeat: 
> The tar problem is likely because git lfs files were not pulled.

If you check the content of those files, you'd be able to ascertain whether that is correct or something else is at play.

**Comment by gaojindev** [comment]
Oh, right.

**Comment by gaojindev** [comment]
I just got them
Thanks
:)

**Comment by superdev712** [comment]
I am facing issues in building this library on windows
Is it possible to use MSVC C/C++ compiler?
I mean I didn't set set CC=clang-cl and set CXX=clang-cl
And it succeeded to generate script with cmake.
But I get 
![image](https://github.com/sighthoundinc/SighthoundVideo/assets/150344472/b0ad6c34-10e0-41ab-b754-792dbbc06a3f)

**Comment by superdev712** [comment]
I will retry after install clang tools in VS but above is related to SVPythonDIR environment path

**Comment by superdev712** [comment]
I installed clang tools but the same error occurs

**Comment by superdev712** [comment]
I am still having isues in building
And as far as I researched it is related the environment variables.
Actually I use make which I installed after installing msys2 and using pacman coomand.
So make uses msys2 system and its variables.
However the native x64 command prompt of VS seems to be using my Windows system and so it cause difference in variables, I think
Is there anyone who solved this issue?

**Comment by superdev712** [comment]
Oh, I managed to solve that issue myself.
Actually I installed make using msys - packman -S make which installs make into default directory - /usr/bin
Thus when I add make to environment variable, it also adds bash of msys to the environment variable.
And that conflicts with bash of git-bash mentioned in documentation.
So I copied make.exe from the default dir to the other directory and add that path to the env variable.
That solved the issue
